recipe/meta.yaml

```diff
@@ -25,18 +22,43 @@ requirements:
     - pip
     - python
     - setuptools
-    - setuptools-rust
     - wheel
+    - maturin 1.3.0
   run:
     - python
+  run_constrained:
+    - numpy >= 1.21.6
+    - tensorflow >= 2.11
+    - jax >= 0.3.25
+    - jaxlib >= 0.3.25
+    - flax >= 0.6.3
 
 test:
+  source_files:
+    # Excluded test modules:
```

Comment:
I would do it the other way around. Include all test files and skip the files that are problematic (`pytest --ignore bindings/python/tests/test_pt_comparison.py ...`). It'll be easier to maintain.